iOS/Second-Hand/Second-Hand/UI/Account/SignUpViewController.swift

```diff
@@ -0,0 +1,146 @@
+                self.idInputView.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor),
+                self.idInputView.trailingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.trailingAnchor),
+                self.idInputView.heightAnchor.constraint(equalToConstant: idInputViewHeight)
+            ]
+        )
+    }
+    
+    private func setupNavigationBarLayoutConstraint() {
+        self.navigationBar.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate(
+            [
+                self.navigationBar.topAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.topAnchor),
+                self.navigationBar.leadingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.leadingAnchor),
+                self.navigationBar.trailingAnchor.constraint(equalTo: self.view.safeAreaLayoutGuide.trailingAnchor)
+            ]
+        )
+    }
+    // swiftlint:disable:next function_body_length
+    private func setupLoactionAddButtonLayoutConstraint() {
+        let locationAddButtonTopPadding: CGFloat = 51
+        let locationAddButtonleadingPadding: CGFloat = 16
+        let locationAddButtonTrailingPadding: CGFloat = -16
+        
+        self.locationAddButton.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate(
+            [
+                self.locationAddButton.topAnchor.constraint(
+                    equalTo: self.idInputView.bottomAnchor,
+                    constant: locationAddButtonTopPadding
+                ),
+                self.locationAddButton.leadingAnchor.constraint(
+                    equalTo: self.view.leadingAnchor,
+                    constant: locationAddButtonleadingPadding
+                ),
+                self.locationAddButton.trailingAnchor.constraint(
+                    equalTo: self.view.trailingAnchor,
+                    constant: locationAddButtonTrailingPadding
+                )
+            ]
+        )
+    }
+    
+    private func setupSheet() {
+        if let sheet = sheetPresentationController {
+            sheet.detents = [ .large(), .medium() ]
+            sheet.selectedDetentIdentifier = .large
+            sheet.largestUndimmedDetentIdentifier = .large
+        }
+    }
+    
+    private func makeNavigationBarItem() {
+        let closeButton = UIBarButtonItem(
+            title: "닫기",
+            style: .plain,
+            target: self,
+            action: #selector(self.tappedBackButton)
+        )
+        let completionButton = UIBarButtonItem(title: "완료", style: .plain, target: self, action: nil)
+        self.navigationItem.leftBarButtonItem = closeButton
+        self.navigationItem.rightBarButtonItem = completionButton
+    }
+    
+    private func setupNavigationBarItem() {
+        self.navigationItem.rightBarButtonItem?.tintColor = ColorPalette.black
+        self.navigationItem.leftBarButtonItem?.tintColor = ColorPalette.black
```

Comment:
```suggestion
        self.navigationItem.rightBarButtonItem?.tintColor = ColorPalette.gray800
        self.navigationItem.leftBarButtonItem?.tintColor = ColorPalette.gray900
```
피그마를 확인하고 오니 색상이 요렇게 되어야 할 것 같습니다!!